src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/MultiprojectReportAggregationFunctionalSpec.groovy

```diff
@@ -0,0 +1,22 @@
+package info.solidsoft.gradle.pitest.functional
+
+import groovy.transform.CompileDynamic
+import nebula.test.functional.ExecutionResult
+
+@CompileDynamic
+class MultiprojectReportAggregationFunctionalSpec extends AbstractPitestFunctionalSpec {
+
+    void "should aggregate report from subproject"() {
+        given:
+            copyResources("testProjects/report", "")
```

Comment:
I wonder, wouldn't be possible to run both multiproject and report tests on the same test project?

Reply:
Yes, its possible. I merged two tests into one

Reply:
As mentioned above, it didn't take into account some complications. Sorry for extra trouble I generated...